salt/transport/tcp.py

```diff
@@ -363,6 +363,7 @@ def post_fork(self, message_handler, io_loop):
         message_handler: function to call with your payloads
         """
         self.message_handler = message_handler
+        log.info("ReqServer workers %s", socket)
```

Comment:
At info level?

Reply:
For this one, yes. We have the same log message in the zeromq transport.